src/utils.ts

```diff
@@ -10,21 +10,20 @@ import parser from 'any-date-parser'
 export const WALLET_FILE_LOCATION = './xmtp_wallet'
 
 export const randomWallet = (): Wallet => {
-  const key = PrivateKey.generate()
-  if (!key.secp256k1) {
-    throw new Error('invalid key')
-  }
-  return new Wallet(key.secp256k1.bytes)
+  return Wallet.createRandom()
```

Comment:
We do it the longer way only because it's something like 100X faster than Wallet.createRandom. It only really matters if you are creating thousands of test inboxes to fill an inbox.